### DISCUSSION TITLE
RasberryPi Bullseye, can I install an older version (Version: 188-1)?

### DISCUSSION THREAD

**Discussion by Telistra** [cockpit-project/cockpit#18228] [General]
I have a need to install and earlier version of cockpit, specificlly Version: 188-1. We just upgraded our Pis to Bullseye and saw there was large changes made to how it looked on Buster and in some cases, how it functioned. 

Is that possible, yes/no?

**Comment by garrett** [comment]
[Cockpit 188](https://cockpit-project.org/blog/cockpit-188.html) was from February 2019, which is almost _4 years old_ at this point in time. There have so been many updates since then, including lots of new features and bugfixes.

If you want to keep running such an old version, then you would need to use a supported long term distribution with Cockpit 188, such as Debian Buster.

However, we recommend using newer versions, not older versions. If you're using Debian and not Raspbian/RaspberryPiOS*, then backports are available too: https://cockpit-project.org/running.html#debian (But this would get you an even newer version, which is the opposite of what you want.)

(Debian is better for servers; Raspberry Pi OS — previously known as Raspbian — is a fork that's intended for children to use a simple desktop, it's not meant to be used as a server. It's similar, but not entirely the same as Debian. There are some compatibility issues with it... I'm speaking from experience, as I've hit some specific Raspbian / Raspberry Pi OS issues before; things like backports will not work properly with it.)

**Reply by ericofplymouth** [reply]
Hi Peter: v 188 of Cockpit is way behind the current look-and-feel. 

I have chosen to use Webmin instead on my Raspberry Pi 4b.  See the docs for Webmin here: https://webmin.com/docs/  I've been very impressed with Webmin and its consistent GUI, through many versions and distros over the years!!!  Well done, Jamie Cameron!  https://webmin.com/about-jamie/.  

Please give Webmin a try and reply with your impressions.

- Eric

**Comment by garrett** [comment]
> large changes made to how it looked on Buster and in some cases, how it functioned

Why do you want to stick with 188 and not upgrade to a newer version?

**Reply by Telistra** [reply]
Mainly because when we installed the latest version, we found issues with some metrics not displaying out of the box, along with some useful features removed, found in the previous versions of Buster. And the question was asked, can we just install the old version. 

We don't use them as servers, and they are hardly just for children, let's be honest here. We use them as headless controllers, for our Aircraft tracking, weather sensor data collection and as a local DMR Repeater for our digital radios using PiStar.
